addons/kommander/1.2/kommander.yaml

```diff
@@ -46,7 +46,7 @@ spec:
   chartReference:
     chart: kommander
     repo: https://mesosphere.github.io/charts/stable
-    version: 0.12.10
+    version: 0.12.12
```

Comment:
oopsies we've since force pushed our branch this is based on so the contents of this file are out of date

Reply:
(fixed)